cranelift/frontend/src/frontend.rs

```diff
@@ -229,6 +229,11 @@ impl<'a> FunctionBuilder<'a> {
         block
     }
 
+    /// Mark a block as cold.
+    pub fn mark_cold_block(&mut self, block: Block) {
```

Comment:
Ditto re `set_cold` here.

Reply:
I renamed this to `set_cold_block` rather than just `set_cold` for consistency with the other method names, e.g. `seal_block` and `switch_to_block`; the builder deals with more than just blocks (as opposed to the block-layout internal API above) so I think this is probably clearer. Thanks!